### PR TITLE
update es env

### PR DESCRIPTION
**Pull request by xuelangos**
SW_STORAGE=elasticsearch  update  ES_VERSION=es7

### CODE REVIEW

File: 7/7.0/oap-es7/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ $ docker run --name oap --restart always -d apache/skywalking-oap-server:7.0.0-e
 ## Start a `standlone` container with `elasticsearch` storage whose address is `elasticsearch:9200`
 
 ```
-$ docker run --name oap --restart always -d -e SW_STORAGE=elasticsearch -e SW_STORAGE_ES_CLUSTER_NODES=elasticsearch:9200 apache/skywalking-oap-server:7.0.0-es7
+$ docker run --name oap --restart always -d -e ES_VERSION=es7  -e SW_STORAGE_ES_CLUSTER_NODES=elasticsearch:9200 apache/skywalking-oap-server:7.0.0-es7
```
Comment (wu-sheng) [inline 7/7.0/oap-es7/README.md:23]: I think this is not correct.

Comment (xuelangos) [reply 7/7.0/oap-es7/README.md:23]: 按照你的写法，直接跑步起来，我的ES版本7.4.2
[Entrypoint] Apache SkyWalking Docker Image
Current image doesn't Elasticsearch 6

Comment (wu-sheng) [reply 7/7.0/oap-es7/README.md:23]: I know the existing doc is not correct, but your change is not correct too.

Comment (xuelangos) [reply 7/7.0/oap-es7/README.md:23]: ok，哪可以改一下正确的值么

**Pull request closed without merging**